packages/devextreme/testing/tests/DevExpress.ui.widgets/menu.tests.js

```diff
@@ -3856,6 +3858,87 @@ QUnit.module('adaptivity: behavior', {
     });
 });
 
+let helper;
+QUnit.module('Aria accessibility', {
+    beforeEach: function() {
+        helper = new ariaAccessibilityTestHelper({});
+        fx.off = true;
+        this.clock = sinon.useFakeTimers();
+    },
+    afterEach: function() {
+        fx.off = false;
+        this.clock.restore();
+    }
+}, () => {
+    QUnit.test('Nested submenu has the "menu" role', function(assert) {
+        if(!isDeviceDesktop(assert)) {
+            return;
+        }
+
+        const menu = createMenu({
+            items: [{
+                text: 'item 1',
+                items: [{
+                    text: 'item 11',
+                    items: [{
+                        text: 'item 111',
+                    }],
+                }],
+            }],
+            showFirstSubmenuMode: 'onClick',
+            showSubmenuMode: { name: 'onHover', delay: 0 },
+        });
+        const $item1 = $(menu.element).find(`.${DX_MENU_ITEM_CLASS}`).eq(0);
+
+        $item1.trigger('dxclick');
+
+        const submenu = getSubMenuInstance($item1);
+
+        const $menuItem = $(submenu._overlay.content()).find(`.${DX_MENU_ITEM_CLASS}`).first();
```

Comment:
submenu has `getOverlayContent` method for such purposes
```suggestion
        const $menuItem = submenu.getOverlayContent().find(`.${DX_MENU_ITEM_CLASS}`).first();
```